packages/tempus/src/Tempus_ODEParameters.hpp

```diff
@@ -0,0 +1,50 @@
+// @HEADER
+// ****************************************************************************
+//                Tempus: Copyright (2017) Sandia Corporation
+//
+// Distributed under BSD 3-clause license (See accompanying file Copyright.txt)
+// ****************************************************************************
+// @HEADER
+
+#ifndef Tempus_ODEParameters_hpp
+#define Tempus_ODEParameters_hpp
+
+
+#include "Tempus_TimeDerivative.hpp"
+#include "Tempus_WrapperModelEvaluator.hpp"
+
+
+namespace Tempus {
+
+
+template<class Scalar>
+class ODEParameters
+{
+  public:
+    /// Constructor
+    ODEParameters()
+      : timeDer_(Teuchos::null), timeStepSize_(Scalar(0.0)), stageNumber_(0),
+        alpha_(Scalar(0.0)), beta_(Scalar(0.0)),
```

Comment:
You could just set default values for the fields where you declare them.  For example:
```c++
Scalar timeStepSize_ (0.0);
int stageNumber_ (0);
```
The default value of an RCP is always as if you assigned `Teuchos::null` to it, so you don't need to specify the default there.

Reply:
Yeah, old habits.  :)